src/features/Instructors/InstructorForm/index.jsx

```diff
@@ -99,12 +101,48 @@ const AddInstructors = ({ isOpen, onClose }) => {
     return null;
   };
 
+  const handleEditInstructor = async (e) => {
+    e.preventDefault();
+    try {
+      const formData = new FormData();
+      formData.append('enrollment_privilege', formState.instructor.hasEnrollmentPrivilege);
+      formData.append('institution_id', selectedInstitution.id);
+      formData.append('instructor_id', instructorInfo.instructorId);
+
+      await dispatch(editInstructor(formData));
+      handleCloseModal();
+      dispatch(updateInstructorFormRequest({ status: RequestStatus.INITIAL }));
+    } catch (error) {
+      logError(error);
+    }
+  };
+
+  const isValidInstructorInfo = (info) => (
```

Comment:
This should go outside the component to avoid being created every time the component mounts.